lib/definitely-typed.ts

```diff
@@ -83,39 +104,21 @@ async function getIndex(content: string, packageName: string): Promise<string> {
     } catch (e) {
         console.warn(`Warning: Could not retrieve author's user name: ${e.message}`);
     }
-    const authorUrl = formatUrl({
-        protocol: 'https',
-        hostname: 'github.com',
-        pathname: authorUserName,
-    });
-
-    return `// Type definitions for ${packageName} ${version}
-// Project: ${project}
-// Definitions by: ${authorName} <${authorUrl}>
-// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-${content}`;
-}
-
-function getTSConfig(dtName: string): {} {
     return {
-        compilerOptions: {
-            module: "commonjs",
-            lib: ["es6"],
-            noImplicitAny: true,
-            noImplicitThis: true,
-            strictFunctionTypes: true,
-            strictNullChecks: true,
-            baseUrl: "../",
-            typeRoots: ["../"],
-            types: [],
-            noEmit: true,
-            forceConsistentCasingInFileNames: true,
+        private: true,
+        name: packageName,
```

Comment:
packagename is something like `a__b` for `@a/b`, but should be `@types/a__b`

Reply:
Hm, seems like packageName here is the name of the non-types package, not the mangled name either.

```ts
const reg: Registry = JSON.parse(await loadString(`https://registry.npmjs.org/${packageName}`));
```

So, I need to mangle it and that's it.

Reply:
Updated, PTAL